setup.py

```diff
@@ -55,7 +55,7 @@
         "eth-keyfile>=0.7.0, <0.9.0",
         "eth-keys>=0.4.0",
         "eth-rlp>=2.1.0",
-        "eth-utils>=2.0.0",
+        "eth-utils>=5.3.0",
```

Comment:
Does this merit a major or minor bump release? Curious what impact this may have.

Reply:
Yeah, historically we've called a lower pin change like this a major. I think we should wait until our breaking cycle, but could be convinced otherwise :)